src/rt/sched/work.h

```diff
@@ -21,6 +21,8 @@ namespace verona::rt
   {
     static constexpr auto NO_EPOCH_SET = (std::numeric_limits<uint64_t>::max)();
 
+    bool yielded = false;
```

Comment:
I don't think this field is needed, and if it is I am not keen to expand all work items for this feature.